dft-libxc/parallel/functionals_dft/rdft-CNm-HCTH-A-gradient.json

```diff
@@ -7,126 +7,126 @@
       "value": "1.0000000000",
       "type": "float",
       "precision": 10,
-      "tolerance": 7.5e-07
+      "tolerance": 5e-08
```

Comment:
I don't think that the tolerances should be tightened.

Reply:
Will fix!

Reply:
addressed in 58d37c75a819e0cd78ca205b47f649cc09166a0f